doc/doxygen/references.bib

```diff
@@ -1766,6 +1766,19 @@ @MastersThesis{Battermann1996
   url       = {http://hdl.handle.net/10919/9579}
 }
 
+@article{Boffi2012,
+  author = {Boffi, D. and Cavallini, N. and Gardini, F. and Gastaldi, L.},
```

Comment:
```suggestion
  author = {D. Boffi and N. Cavallini and F. Gardini and L. Gastaldi},
```

In #12487, we agreed on a standardized format for author names. Would you update this part correspondingly?